pkg/controller/dashboards/netstats.json

```diff
@@ -0,0 +1,1495 @@
+                        "name":null,
+                        "show":true,
+                        "values":[
+                        ]
+                    },
+                    "yaxes":[
+                        {
+                            "format":"Bps",
+                            "label":null,
+                            "logBase":1,
+                            "max":null,
+                            "min":null,
+                            "show":true
+                        },
+                        {
+                            "format":"Bps",
+                            "label":null,
+                            "logBase":1,
+                            "max":null,
+                            "min":null,
+                            "show":true
+                        }
+                    ]
+                },
+                {
+                    "aliasColors":{
+                    },
+                    "bars":false,
+                    "dashLength":10,
+                    "dashes":false,
+                    "datasource":"prometheus",
+                    "fill":10,
+                    "fillGradient":0,
+                    "gridPos":{
+                    },
+                    "id":1,
+                    "legend":{
+                        "alignAsTable":false,
+                        "avg":false,
+                        "current":false,
+                        "max":false,
+                        "min":false,
+                        "rightSide":false,
+                        "show":true,
+                        "sideWidth":null,
+                        "total":false,
+                        "values":false
+                    },
+                    "lines":true,
+                    "linewidth":1,
+                    "links":[
+                    ],
+                    "nullPointMode":"null",
+                    "percentage":false,
+                    "pointradius":5,
+                    "points":false,
+                    "renderer":"flot",
+                    "repeat":null,
+                    "seriesOverrides":[
+                    ],
+                    "spaceLength":10,
+                    "span":6,
+                    "stack":true,
+                    "steppedLine":false,
+                    "targets":[
+                        {
+                            "expr":"sum(label_replace(label_replace(rate(container_network_transmit_bytes_total[1m]), 'dev', '$1', 'interface', '(.*)'), 'dev', '(pod)', 'interface', '[a-f0-9]{15}')) by (dev)",
+                            "format":"time_series",
+                            "intervalFactor":2,
+                            "legendFormat":"{{dev}}",
+                            "refId":"A"
+                        }
+                    ],
+                    "thresholds":[
+                    ],
+                    "timeFrom":null,
+                    "timeShift":null,
+                    "title":"Bytes Sent, by Device",
```

Comment:
It can be any linux network interface right? I _think_ `interface` is better than device.

Reply:
yes correct. I was reusing the node-exporter terminology (they use "device" or "dev") but I imagine "interface" is more correct - that's also what we use in netobserv